src/component/Table.jsx

```diff
@@ -22,9 +22,9 @@ function Table({ save, setSave }) {
       <table>
         <thead>
           <tr>
-            <td>Item</td>
-            <td>Packed</td>
-            <td>Note</td>
+            <td>{column1}</td>
```

Comment:
This should be dynamic, what if you have more than 3 columns? Try to pass down an array of items and use map to render all no matter how many they are.